packages/manager/apps/catalog/e2e/step-definitions/error.step.ts

```diff
@@ -0,0 +1,60 @@
+import { Given, When, Then } from '@cucumber/cucumber';
+import { expect } from '@playwright/test';
+import { ICustomWorld, sleep } from '../../../../../../playwright-helpers';
```

Comment:
Couldn't we set some path alias to avoid thoses very long relative path ?

Reply:
No it doesn't work